MapboxCoreNavigation/CLLocation.swift

```diff
@@ -159,16 +163,16 @@ extension CLLocation {
     }
     
     /**
-     Determines if the a location is qualified enough to allow the user puck to become unsnapped.
+     Determines if the course of a location is qualified enough to allow the user puck to be snapped
      */
-    func shouldSnap(toRouteWith course: CLLocationDirection, distanceToFirstCoordinateOnLeg: CLLocationDistance = CLLocationDistanceMax) -> Bool {
+    func shouldSnapCourse(toRouteWith course: CLLocationDirection, distanceToFirstCoordinateOnLeg: CLLocationDistance = CLLocationDistanceMax) -> Bool {
```

Comment:
Is there a reason for renaming the function?

Reply:
Yes, because this function is only checking the course. The name was misleading before. We could also do the distance check in this function, but this would require to calculate it again (as this is done in a different piece of the code).